src/main/pages/che-6/user-guide/creating-starting-workspaces.adoc

```diff
@@ -10,41 +10,59 @@ folder: che-6/user-guide
 
 
 [id="creating-workspaces"]
-== Creating Workspaces
+== Creating workspaces
 
-The easiest way to create a workspace is to use ready to go link:stacks.html[stacks] in User Dashboard. Images and configuration in these stacks is certified both for Docker and OpenShift, and these are the stacks that are used in daily functional testing. There are multiple options though, namely:
+Use the ready-to-go link:stacks.html[stacks] in the user's *Dashboard* to create a workspace. Images and configuration in these stacks are certified both for Docker and OpenShift. These stacks are used in daily functional testing.
 
-*Ready-to-go Stacks*
+=== Creating a workspace from ready-to-go stacks
```

Comment:
Wonder if it makes sense to have instructions/steps in addition to these gifs.

Reply:
Added. Thanks :)